src/main/java/gregtech/common/metatileentities/multi/electric/generator/MetaTileEntityLargeTurbine.java

```diff
@@ -86,6 +86,7 @@ protected void formStructure(PatternMatchContext context) {
 
     @Override
     public void invalidateStructure() {
+        getAbilities(ABILITY_ROTOR_HOLDER).get(0).resetRotorSpeed();
```

Comment:
Please make public function in class which will provide rotor so that it is easily accessible